recipes/systemc/2.3.3/conanfile.py

```diff
@@ -0,0 +1,86 @@
+    homepage = "https://www.accellera.org/"
+    url = "https://github.com/conan-io/conan-center-index "
+    license = "MIT"
+    topics = ("simulation", "modeling", "esl", "tlm")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "disable_async_updates": [True, False],
+        "disable_copyright_msg": [True, False],
+        "disable_virtual_bind": [True, False],
+        "enable_assertions": [True, False],
+        "enable_immediate_self_notifications": [True, False],
+        "enable_pthreads": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "disable_async_updates": False,
+        "disable_copyright_msg": False,
+        "disable_virtual_bind":  False,
+        "enable_assertions": True,
+        "enable_immediate_self_notifications": False,
+        "enable_pthreads": False
+    }
+    generators = "cmake"
+    exports_sources = "patches/**"
+
+    def configure(self):
+
+        if self.settings.os == "Macos":
+            raise ConanInvalidConfiguration("Macos build not supported")
+
+        if tools.valid_min_cppstd(self, "17"):
+            raise ConanInvalidConfiguration(
+                "C++ Standard %s not supported by SystemC" %
+                self.settings.compiler.cppstd)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        if self.settings.compiler.cppstd:
+            cmake.definitions["CMAKE_CXX_STANDARD"] = \
+                    self.settings.compiler.cppstd
+        cmake.definitions["DISABLE_ASYNC_UPDATES"] = \
+            self.options.disable_async_updates
+        cmake.definitions["DISABLE_COPYRIGHT_MESSAGE"] = \
+            self.options.disable_copyright_msg
+        cmake.definitions["DISABLE_VIRTUAL_BIND"] = \
+            self.options.disable_virtual_bind
+        cmake.definitions["ENABLE_ASSERTIONS"] = \
+            self.options.enable_assertions
+        cmake.definitions["ENABLE_IMMEDIATE_SELF_NOTIFICATIONS"] = \
+            self.options.enable_immediate_self_notifications
+        cmake.definitions["ENABLE_PTHREADS"] = self.options.enable_pthreads
+        cmake.configure(source_folder="systemc-{}".format(self.version))
+        return cmake
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(base_path="systemc-{}".format(self.version), **patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir("{}/lib/cmake".format(self.package_folder))
+        tools.rmdir("{}/share".format(self.package_folder))
+        self.copy("LICENSE", dst="licenses",
+                  src="systemc-{}".format(self.version))
+
+    def package_info(self):
+        self.cpp_info.libs = ["systemc"]
```

Comment:
Can you check the name of the `cmake` modules that you remove in `package`?
You should do something like this here:
```python
self.deps_cpp_info.names["cmake_find_package"] = "SystemC"
self.deps_cpp_info.names["cmake_find_package_multi"] = "SystemC"
```

Reply:
I will just add `SystemC` the target used in the `cmake` dir is actually `SystemC:: systemc` but I cant find any info on who to pass this to conan

Reply:
conan components (and cmake_find_package) integration is a WIP. See https://github.com/conan-io/conan/pull/6871
In my recipes, I would add a `#FIXME: cmake generates SystemC::systemc target` as a reminder to the user that this is missing.

Reply:
Thx for the suggestion, updated accordingly